core-types/nested_collections.rb

```diff
@@ -0,0 +1,96 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+
+class NestedCollectionsTest < Minitest::Test
+  
+  def test_2D_array
+    skip
+    numbers = [[1, 2], [9, 4, 89], 9]
+    
+    assert_equal __, numbers.first
+    assert_equal __, numbers[0][1]
+    assert_equal __, numbers[1][3]
```

Comment:
Just double checking that you meant to call an index position that isn't in the array.

Reply:
Yes, just to give an example of what happens when you do; is that too confusing?